src/arch/common/forte_specific_architecture.cpp

```diff
@@ -1,27 +1,21 @@
 /*******************************************************************************
- * Copyright (c) 2017 fortiss GmbH
+ * Copyright (c) 2024 
```

Comment:
please put your name as the copyright holder of this file

Reply:
done